clang/include/clang/Format/Format.h

```diff
@@ -265,6 +279,7 @@ struct FormatStyle {
              AcrossComments == R.AcrossComments &&
              AlignCompound == R.AlignCompound &&
              AlignFunctionPointers == R.AlignFunctionPointers &&
+             AlignFunctionDeclarations == R.AlignFunctionDeclarations &&
```

Comment:
D before P? we try to sort the alphabetically

Reply:
sorry, missed that.  Done.